### PR TITLE
Gcm driver data munge pipeline

### PR DESCRIPTION
**Pull request by hcorson-dosch**
Merge the `gcm_driver_data_munge_pipeline` into the main branch of the canonical repo.

This branch sets up a `targets` sub-pipeline within the `scipiper` `lake_temperature-model-prep` repository that:
* Reconstructs the 25-km resolution grid associated with the downscaled GCM driver data
* Constructs tiles that are 15 cells wide by 15 cells high for the purpose of bundling the GDP queries
* Determines which cells contain lakes
* For each tile and each GCM (6 in total), queries GDP to download the downscaled GCM data only for grid cells within that tile that contain lakes
  * Uses cell centroids in GDP query, converted to simplegeom polygons
  * Queries GDP for 6 variables (precipitation, air temperature, relative humidity, surface shortwave radiation, surface longwave radiation, and windspeed)
* Munges the downloaded data for use with GLM
* _(temporary)_ Writes the munged data to feather files to be used in `lake-temperature-process-models`
* Writes all of the data for all GCM cells to a single netCDF file for each GCM

This `targets` sub-pipeline is connected to the full scipiper pipeline and can be kicked off by running `scmake('7_drivers_munge/out/7_GCM_driver_files.ind')`.

### CODE REVIEW

File: .gitignore
Comment (lindsayplatt) [inline .gitignore:59]: We need to ignore pngs within the 7_drivers_munge, not everywhere

File: .gitignore
Comment (lindsayplatt) [inline .gitignore:42]: No need to have this. We don't have an "in" folder

**Review by lindsayplatt — APPROVED** [review]
Two small gitignore changes and then good! All of this has been reviewed in separate PRs to the `gcm_driver_data_munge_pipeline`.

**Comment by hcorson-dosch** [comment]
Don't merge until merge #305

**Pull request merged**
